### ISSUE TITLE
Service does not start on 0.8.11

### ISSUE THREAD

**Issue by fmunteanu** [PowerDataHub/terraform-aws-airflow#20]
I managed to deploy the cluster using examples\deploy, however, the service does not start:
```
Jul 18 17:26:47 ip-172-31-40-22 systemd[1]: Started Airflow daemon.
Jul 18 17:26:47 ip-172-31-40-22 terraform-aws-airflow[25465]: [2019-07-18 17:26:47,651] {settings.py:182} INFO - settings.configure_orm(): Using pool settings. pool_size=5, pool_recycle=1800, pid=25465
Jul 18 17:26:47 ip-172-31-40-22 terraform-aws-airflow[25465]: [2019-07-18 17:26:47,879] {__init__.py:51} INFO - Using executor CeleryExecutor
Jul 18 17:26:48 ip-172-31-40-22 terraform-aws-airflow[25465]:   ____________       _____________
Jul 18 17:26:48 ip-172-31-40-22 terraform-aws-airflow[25465]:  ____    |__( )_________  __/__  /________      __
Jul 18 17:26:48 ip-172-31-40-22 terraform-aws-airflow[25465]: ____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
Jul 18 17:26:48 ip-172-31-40-22 terraform-aws-airflow[25465]: ___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
Jul 18 17:26:48 ip-172-31-40-22 terraform-aws-airflow[25465]:  _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
Jul 18 17:26:48 ip-172-31-40-22 terraform-aws-airflow[25465]: [2019-07-18 17:26:48,581] {__init__.py:305} INFO - Filling up the DagBag from /dev/null
Jul 18 17:26:49 ip-172-31-40-22 terraform-aws-airflow[25465]: [2019-07-18 17:26:49,148] {security.py:446} INFO - Start syncing user roles.
Jul 18 17:26:49 ip-172-31-40-22 terraform-aws-airflow[25465]: [2019-07-18 17:26:49,624] {security.py:357} INFO - Fetching a set of all permission, view_menu from FAB meta-table
Jul 18 17:26:49 ip-172-31-40-22 terraform-aws-airflow[25465]: [2019-07-18 17:26:49,963] {security.py:307} INFO - Cleaning faulty perms
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]: [2019-07-18 17:26:50,633] {settings.py:182} INFO - settings.configure_orm(): Using pool settings. pool_size=5, pool_recycle=1800, pid=25507
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]: /usr/local/lib/python3.6/dist-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   """)
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]: [2019-07-18 17:26:50 +0000] [25507] [INFO] Starting gunicorn 19.9.0
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]: Traceback (most recent call last):
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/local/bin/gunicorn", line 11, in <module>
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     sys.exit(run())
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/local/lib/python3.6/dist-packages/gunicorn/app/wsgiapp.py", line 61, in run
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/local/lib/python3.6/dist-packages/gunicorn/app/base.py", line 223, in run
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     super(Application, self).run()
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/local/lib/python3.6/dist-packages/gunicorn/app/base.py", line 72, in run
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     Arbiter(self).run()
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/local/lib/python3.6/dist-packages/gunicorn/arbiter.py", line 199, in run
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     self.start()
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/local/lib/python3.6/dist-packages/gunicorn/arbiter.py", line 139, in start
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     self.pidfile.create(self.pid)
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/local/lib/python3.6/dist-packages/gunicorn/pidfile.py", line 36, in create
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     fd, fname = tempfile.mkstemp(dir=fdir)
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/lib/python3.6/tempfile.py", line 483, in mkstemp
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     return _mkstemp_inner(dir, prefix, suffix, flags, output_type)
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:   File "/usr/lib/python3.6/tempfile.py", line 401, in _mkstemp_inner
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]:     fd = _os.open(file, flags, 0o600)
Jul 18 17:26:50 ip-172-31-40-22 terraform-aws-airflow[25465]: PermissionError: [Errno 13] Permission denied: '/usr/local/airflow/tmpqk3e9n1v'
```

**Comment by edbizarro** [comment]
Hi @fmunteanu 

Before you deploy the code, the services were running normally?

**Comment by fmunteanu** [comment]
@edbizarro I did not deploy any code, I simply ran the following commands:
```
$ git clone --branch v0.8.11 git@github.com:PowerDataHub/terraform-aws-airflow.git
$ cd terraform-aws-airflow/examples/deploy
$ cat main.tf
provider "aws" {
  region = "us-east-1"
}

data "aws_vpc" "default" {
  default = true
}

module "airflow_cluster" {
    source             = "../../"
    cluster_name       = "test"
    cluster_stage      = "dev"
    s3_bucket_name     = "test"
    db_dbname          = "airflow"
    db_username        = "airflow"
    db_password        = "somepassword"
    fernet_key         = "somekey"
    key_name           = "test"
    vpc_id             = "${data.aws_vpc.default.id}"
    load_example_dags  = true
    rbac               = true
    admin_name         = "Floren"
    admin_lastname     = "Munteanu"
    admin_email        = "floren@domain.com"
    admin_username     = "admin"
    admin_password     = "someadminpassword"
}

$ terraform init
$ terraform plan -out dev-compute
$ terraform apply dev-compute
```

**Comment by edbizarro** [comment]
ah ok, I'm looking into that

**Comment by edbizarro** [comment]
I could not reproduce this problem, can you provide your full `/var/log/cloud-init-output.log` 
of any of the machines?

Thanks

**Comment by fmunteanu** [comment]
I found the issue. If there is an ampersand used, the script will fail. For example, my `admin_password` was `m4n-&LE5-fgr7YdZ` and the script was trying to execute the `LE5-fgr7YdZ` command:
```
Jul 19 14:55:34 ip-172-31-40-168 cloud-init[1829]: + airflow upgradedb
Jul 19 14:55:36 ip-172-31-40-168 cloud-init[1829]: [2019-07-19 14:55:36,694] {settings.py:182} INFO - settings.configure_orm(): Using pool settings. pool_size=5, pool_recycle=1800, pid=25044
Jul 19 14:55:37 ip-172-31-40-168 cloud-init[1829]: [2019-07-19 14:55:37,148] {__init__.py:51} INFO - Using executor CeleryExecutor
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: DB: postgresql+psycopg2://airflow:***@eg-ailabs-dev-airflow-db.c7sscm9tzb0l.us-east-1.rds.amazonaws.com:5432/airflow
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: [2019-07-19 14:55:38,035] {db.py:350} INFO - Creating tables
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: /usr/local/lib/python3.6/dist-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]:   """)
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: INFO  [alembic.runtime.migration] Will assume transactional DDL.
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: + '[' true = true ']'
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: + LE5-fgr7YdZ
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: + airflow create_user -r Admin -u admin -f Floren -l Munteanu -e floren@expedia.com -p hRC-
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: /var/lib/cloud/instance/scripts/part-001: line 105: LE5-fgr7YdZ: command not found
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: Cloud-init v. 18.4-0ubuntu1~18.04.1 running 'modules:final' at Fri, 19 Jul 2019 14:50:41 +0000. Up 35.32 seconds.
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: 2019-07-19 14:55:38,328 - util.py[WARNING]: Failed running /var/lib/cloud/instance/scripts/part-001 [127]
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: 2019-07-19 14:55:38,363 - cc_scripts_user.py[WARNING]: Failed to run module scripts-user (scripts in /var/lib/cloud/instance/scripts)
Jul 19 14:55:38 ip-172-31-40-168 cloud-init[1829]: 2019-07-19 14:55:38,383 - util.py[WARNING]: Running module scripts-user (<module 'cloudinit.config.cc_scripts_user' from '/usr/lib/python3/dist-packages/cloudinit/config/cc_scripts_user.py'>) failed
```

**Comment by edbizarro** [comment]
I just published the [v0.8.12](https://github.com/PowerDataHub/terraform-aws-airflow/releases/tag/v0.8.12) tag with this fix

**Comment by fmunteanu** [comment]
Thanks!
